### ISSUE TITLE
Cannot find module 'react-test-renderer/packages' from 'ReactSixteenAdapter.js'

### ISSUE THREAD

**Issue by bentranmyriota** [airbnb/enzyme#2051]
### Current behavior
Tests are failing due to the below error. 
```
Cannot find module 'react-test-renderer/package' from 'ReactSixteenAdapter.js'

    However, Jest was able to find:
    	'./ReactSixteenAdapter.js'
```
### Your environment
```
  System:
    OS: Linux 4.15 Ubuntu 18.04.2 LTS (Bionic Beaver)
    CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
    Memory: 2.44 GB / 15.11 GB
    Container: Yes
    Shell: 4.4.19 - /bin/bash
  Binaries:
    Node: 8.15.1 - /usr/bin/node
    npm: 6.4.1 - /usr/bin/npm
  Browsers:
    Chrome: 72.0.3626.121
    Firefox: 65.0.1
  npmPackages:
    react: ^16.4.0 => 16.8.4 
    react-dom: ^16.4.0 => 16.8.4 
    react-test-renderer: ^16.8.4 => 16.8.4 
```

#### Version

| library             | version
| ------------------- | -------
| enzyme              |  ^3.3.0
| react               | ^16.4.0
| react-dom           |  ^16.4.0
| react-test-renderer | ^16.8.4
| adapter (below)     | 

#### Adapter

- [ -] enzyme-adapter-react-16

**Comment by heartlash** [comment]
Do you have a jestsetup file?
It should have something like this inside it:

``` javascript
import Adapter from 'enzyme-adapter-react-16';

 Enzyme.configure({ adapter: new Adapter() });
```

**Comment by bentranmyriota** [comment]
Yes I do.

**Comment by ljharb** [comment]
Duplicate of #2046. Fixed in v1.11.1.

It's also caused by having a broken jest config - ie, by not including `json` in your moduleFileExtensions.
